cockroachcloud/cloud-org-sso.md

```diff
@@ -30,31 +30,50 @@ Cloud Organization SSO allows you to customize your SSO configuration to meet yo
 
-If you [require SSO authentication](configure-cloud-org-sso.html#require-sso), then when you deprovision a member from your IdP, they can no longer access your {{ site.data.products.db }} organization and there is no need to manually remove them from your {{ site.data.products.db }} organization.
+If you [require SSO authentication](configure-cloud-org-sso.html#require-sso) and disable password authentication, then when you deprovision a member from your IdP, they can no longer access your {{ site.data.products.db }} organization and there is no need to manually remove them from your {{ site.data.products.db }} organization, but some organizations choose to do so.
 
-{{site.data.alerts.callout_success}}
-If your organization includes members whose identity you don't manage, such as partners or consultants, you can leave password authentication enabled for those members, while instructing your internal users to sign in using SSO. Members who sign in using a password must be removed from your {{ site.data.products.db }} organization manually.
+If you require automatic deprovisioning or other automated user management features, refer to [SCIM autoprovisioning](#scim-autoprovisioning). Cockroach Labs does not recommend enabling both SSO autoprovisioning and SCIM autoprovisioning for the same authentication method.
+
+### SCIM autoprovisioning
+
+[System for Cross-Domain Identity Management SCIM](https://www.rfc-editor.org/rfc/rfc7644) autoprovisioning centralizes and automates provisioning and deprovisioning of {{ site.data.products.db }} organization users from your IdP.
+
+Rather than a self-service approach, SCIM autoprovisioning tasks are performed by IdP admins. Cockroach Labs does not recommend enabling both [SSO autoprovisioning](#sso-autoprovisioning) and SCIM autoprovisioning for the same authentication method.
+
+1. In {{ site.data.products.db }}, an admin creates a [service account](console-access-management.html#service-accounts) and grants it the `ADMIN` role.
+1. In your IdP, an admin creates a SCIM application and configures it to use the service account to authenticate to the {{ site.data.products.db }} API.
+1. In your IdP, when an admin assigns or unassigns users or groups to the SCIM application, the SCIM application provisions, updates, or deprovisions users using the {{ site.data.products.db }} API.
+  - New users are notified by email with instructions for signing in to {{ site.data.products.db }}.
+  - Deprovisioned users can no longer sign in, even if their accounts still exist in your IdP.
+
+{{site.data.alerts.callout_info}}
+Operations on a SCIM group are applied to each member of the group individually by the {{ site.data.products.db }} API. For example, assigning a SCIM group to the SCIM application provisions accounts for each of the SCIM group's members.
```

Comment:
I'm not sure about this pair of sentences.
* I would avoid using the phrase "SCIM group" in favor of IDP group
* There's a subtle distinction at least in Okta between assigning a group to an application (there by giving everyone in the group access to login app) and pushing the group to CC. So if the group is both assigned to the app and is pushed then yes the user would automatically get created.

Reply:
OK, but "pushing the group to CC" doesn't create a group object in CC, right? I think Abhinav explained to me that a group is always resolved to the list of members and the operation happens separately to each group member. Do I have it wrong?

Reply:
Basically if the group is only pushed, then the group is created in CC but not the users. If assigned to the app integration and also pushed, then members are automatically created